cmd/juju/application/refresh_test.go

```diff
@@ -937,6 +937,35 @@ func (s *RefreshSuccessStateSuite) TestCharmPath(c *gc.C) {
 	s.assertLocalRevision(c, 42, myriakPath)
 }
 
+func (s *RefreshSuccessStateSuite) TestCharmPathNotFound(c *gc.C) {
+	myriakPath := filepath.Join(c.MkDir(), "riak")
+	_, err := os.Stat(myriakPath)
+	c.Assert(err, gc.ErrorMatches, ".*no such file or directory")
+	_, err = s.runRefresh(c, s.cmd, "riak", "--path", myriakPath)
+	c.Assert(err, gc.ErrorMatches, "file does not exist")
```

Comment:
Would it be helpful to include the path in the error message, so it's ultra-clear which file it's referring to? For example:

```
$ juju refresh tiny-bash --path ./non/existent
ERROR file does not exist: "./non/existent"

# or maybe

$ juju refresh tiny-bash --path ./non/existent
ERROR file "./non/existent" does not exist
```

Compare the standard Go error when (say) opening a file that doesn't exist:

```
./t nonexist
2022/08/10 14:30:59 open nonexist: no such file or directory
```